src/main/java/baseball/domain/Ball.java

```diff
@@ -0,0 +1,25 @@
+package baseball.domain;
+
+public class Ball {
+    private final int maximumNumber = 9;
+    private final int minimumNumber = 1;
+    private int number;
```

Comment:
final 선언 시 대문자로 작성하시는 건 어떨까요 !?
```suggestion
    private final int MAX_NUMBER = 9;
    private final int MIN_NUMBER = 1;
```

Reply:
가르쳐주셔서 감사합니다 ㅠㅠ 정말 기본적인 부분인데 놓쳤는데 정말 아쉽네요 다음 미션부터는 주의해야겠네요